src/content/en/tools/chrome-devtools/console/log.md

```diff
@@ -217,6 +217,7 @@ The browser logs messages to the Console, too. This usually happens when there's
 1. Click the **Log Levels** dropdown and enable the **Verbose** option if it's disabled. You'll learn more
    about filtering in the next section. You need to do this to make sure that the next message you log
    is visible.
+   **Note:** If the Default Levels dropdown is disabled, you may need to close the Console Sidebar. Filter by Message Source below for more information about the Console Sidebar.
```

Comment:
Please break lines at 80 chars

Reply:
Of course, thank you for pointing it out! I added a line break based on the lines above the new ones. Even so, the console printed a few warnings. Is this normal?
<img width="1439" alt="Screen Shot 2020-06-19 at 4 20 43 PM" src="https://user-images.githubusercontent.com/42987854/85173133-e7871300-b248-11ea-9748-8a283681a0cc.png">